templates/email/oxfordshire/_alert_update_after_updates.txt

```diff
@@ -0,0 +1,2 @@
+Watch video Cllr Tim Bearder explains when we don’t fix things
```

Comment:
This is good, although in the text part single newlines will be smushed together in a text email into one paragraph, so you might want to have "Watch video Cllr Tim Bearder explains when we don’t fix things: https://youtu.be/hae9g170sA4"

Also, you need to include this template in the default/alert-update.txt file :) (also with the TRY/CATCH/END).